source/fundamentals/logging.txt

```diff
@@ -10,216 +10,21 @@ Logging
    :depth: 1
    :class: singlecols
 
-Overview
---------
+.. note::
 
-This guide shows you how to configure the driver logger which is responsible
-for reporting events that occur while your application is running. You can
-set the logger level, specify logging settings for individual classes, and
-provide a custom logger implementation.
+  The driver doesn't use the logger in this version. Attempting to use
+  prior settings in this version won't print anything. Instead, see our
+  :doc:`command monitoring guide <monitoring/command-monitoring>`. 
 
-You should read this guide if you need to customize the default behavior of the
-driver logger or want to explore the information provided by the logger.
+Temporary Alternative
+---------------------
 
-Log Levels
-----------
+The Node.js team is currently working on rewriting the logger. In the
+meantime, you can log monitor events in the console using the following snippet:
```

Comment:
```suggestion
meantime, you can output monitor events in the console using the following snippet:
```
Suggestion: I think saying "log monitor" is a little confusing as the distinction between "log" and "monitor" is super blurry. I think it is best if we try to separate the language between these two parts of the driver as much as possible to avoid confusing people.